bench/README.md

```diff
@@ -18,40 +28,214 @@ pip3 install -r bench/requirements.txt
 
 ## Preparation
 
+* Copy local accounts keys to folder `~/.dcl/keyring-test`:
+    ```bash
+    cp ./.localnet/node0/keyring-test/* ~/.dcl/keyring-test
+    ```
```

Comment:
This instruction is valid only for running load tests against localnet. For generic use case you should specify only where you should put trustee keys

Reply:
I should remove from generic use case?

Reply:
Done! ✅